tslint.json

```diff
@@ -132,7 +132,7 @@
         ]
       }
     ],
-    "no-any": true,
+    "no-any": false,
```

Comment:
@victorschinaider I think it is important use this rule, we should use strict typescript